argusclient/client.py

```diff
@@ -18,7 +18,7 @@
 try:
     import http.client as httplib  # Python 3
 except ImportError:
-    import httplib  # Python 2
+    import http.client  # Python 2
```

Comment:
How is this going to help Python 2? Won't code references to `httplib` fail?

Reply:
Hari, Replaced it with `urllib` from `six.moves`